### DISCUSSION TITLE
Personal Version of Fusion360 - No admin installer

### DISCUSSION THREAD

**Discussion by philjmaier** [cryinkfly/Autodesk-Fusion-360-for-Linux#509] [Q&A]
Hi there, I followed your youTube video (around 15:41 minute mark), albeit from 3 years ago, all was going well until the step where you get the admin install exe from the Autodesk site. Personal version doesn't have such a thing, it's a launcher. I presume this can't be installed in this fashion with Wine. Or can it? [https://youtu.be/-BktJspJKgs?list=PLzwMdS5iu_BIsO6RTy7Hy1MbzLMrQE2xe](url)

**Comment by philjmaier** [comment]
Forgot to mention, using Linux Mint 22 and Wine 9.0.

**Comment by CptBrandyTeache** [comment]
is that why I can't get a .exe file from the autodesk site?? I also just got on mint 22 and fusion is the only program i planned to use that doesn't have a linux version
